Cargo.toml

```diff
@@ -5,9 +5,11 @@ authors = ["Raphael Nestler <raphael.nestler@gmail.com>"]
 
 [dependencies]
 rand = "*"
-piston ="0.5"
+piston = "0.5"
 piston2d-graphics = "0.4"
 piston2d-opengl_graphics = "0.6"
+bincode = "0.4"
+rustc-serialize = "0.3"
```

Comment:
You should probably use semantic versioning (prefix all version numbers with a `^`, add a version number to `rand`).